modules/framework/galasa-parent/dev.galasa.framework.auth.spi/build.gradle

```diff
@@ -31,7 +31,14 @@ dependencies {
     testFixturesImplementation 'javax.validation:validation-api'
 }
 
-protobuf {
+protobuf {  
+  // For some reason, the combination of the java test fixtures plugin causes a need to have
+  // the following directories exist before vscode will build the project locally.
+  // So lets add them at the same time we generate the protobuf code...
+  // They can be empty, they just have to exist to satisfy vscode's demands.
+  mkdir "$buildDir/generated/source/proto/testFixtures/java"
```

Comment:
This solved my vscode problem in the end, though I still have to re-start the language server every time the build-locally.sh script runs for some reason.